eth/beacon/types/crosslink_record.py

```diff
@@ -0,0 +1,34 @@
+import rlp
+
+from eth.constants import (
+    ZERO_HASH32,
+)
+from eth.rlp.sedes import (
+    int64,
+    hash32,
+)
+
+
+class CrosslinkRecord(rlp.Serializable):
+    """
+    Note: using RLP until we have standardized serialization format.
+    """
+    fields = [
+        # What dynasty the crosslink was submitted in
+        ('dynasty', int64),
+        # slot during which crosslink was added
+        ('slot', int64),
+        # The block hash
+        ('hash', hash32),
+    ]
+
+    def __init__(self,
+                 dynasty=0,
+                 slot=0,
```

Comment:
There is a set of "genesis" crosslinks -- 1 for each shard -- that are inserted into the genesis state. These params match those genesis params.

That said, that should probably happen in an explicit separate process rather than be applied as defaults to each.